src/ansys/units/_constants.py

```diff
@@ -23,6 +23,11 @@ class _QuantityType:
 
 _multipliers: dict = qc_data["multipliers"]
 _unit_systems: dict = qc_data["unit_systems"]
-_api_quantity_map: dict = qc_data["api_quantity_map"]
 _base_units: dict = qc_data["base_units"]
 _derived_units: dict = qc_data["derived_units"]
+
+map_path = os.path.join(file_dir, "quantity_maps/si_map.yaml")
```

Comment:
I think we need to decide how to make this choice configurable.

Reply:
Will be discussed further and done in future work.